CONSTRUCT.md

```diff
@@ -113,6 +113,18 @@ for example, if `python=3.6` is included, then conda will always seek versions
 of packages compatible with Python 3.6. If this is option is not provided, it
 will be set equal to the value of `specs`.
 
+### `virtual_specs`
+
+_required:_ no<br/>
+_type:_ list<br/>
+
+A list of virtual packages that must be satisfied at install time. Virtual
+packages must start with `__`. For example, `__osx>=11` or `__glibc>=2.24`.
+These specs are dry-run solved offline by the bundled `--conda-exe` binary.
```

Comment:
Is the condo exe compatible with a much lower bound than one would otherwise expect. Say 2.12 on Linux.

Reply:
Is seems that on Linux you use ldd but that on osx you use a dry run. 


any way to also use shell commands on osx? I feel like that would be more robust to increases in minimum requirements in osx

Reply:
It's there:

https://github.com/conda/constructor/pull/809/files#diff-d5126ee7f99e6583ba9574579ff73994eaf0eebcfdc792a6deb00a0dc2158f6eR24-R35

And also for PKG:

https://github.com/conda/constructor/pull/809/files#diff-620cc606622328c2ee309e832c215dbf23da87df82a5b10bd09ced66c2f9b503R193-R203

Reply:
amazing, thanks!